bases/br_ibge_pof/caracteristicas_dieta_2017/table_config.yaml

```diff
@@ -0,0 +1,573 @@
+    - name: V71052
+      bigquery_type: string
+      description: Identifica se a dieta foi para pressão alta
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V71053
+      bigquery_type: string
+      description: Identifica se a dieta foi para colesterol
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V71054
+      bigquery_type: string
+      description: Identifica se a dieta foi para diabetes
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V71055
+      bigquery_type: string
+      description: Identifica se a dieta foi para doença no coração
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V71056
+      bigquery_type: string
+      description: Identifica se a dieta foi para outra necessidade
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V71A01
+      bigquery_type: string
+      description: Identifica se a informante está grávida
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V71A02
+      bigquery_type: string
+      description: Identifica se a informante está amamentando
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V72C01
+      bigquery_type: int64
```

Comment:
é int64 mesmo? porque a variável `quantidade_final` que está em `kg` da tabela `caderneta_coletiva_2017` está como `float64`. O ideal é padronizar, o que for `kg` ser sempre `float64`, por exemplo. `int64`, no geral, é utilizado para quantidades em unidades.

Reply:
também achei estranho, mas o peso está em números inteiros msm